manipulator_gym/utils/gym_wrappers.py

```diff
@@ -5,6 +5,59 @@
 import logging
 
 
+class CheckAndRebootJoints(gym.Wrapper):
+    """
+    Every step, check whether joints have failed and reboot them if necessary.
+    """
+    def __init__(self, env, interface, check_every_n_steps: int = 1):
```

Comment:
i wonder what is the overhead/latency of checking the motor every `step(action)` fn?

if it introduces significant overhead, should we have a higher default value, e.g. 10 ?

Reply:
In my runs I think they were pretty fast. I was just eyeballing it though (so on a precision of 1/0.1 seconds?), did not actually time it

Reply:
nice, good to know that it is fast enough =)